kong/clustering/init.lua

```diff
@@ -53,22 +64,31 @@ function _M:init_cp_worker(plugins_list)
 
   events.init()
 
+  self.json_handler:init_worker(plugins_list)
   self.wrpc_handler:init_worker(plugins_list)
 end
 
+
 function _M:init_dp_worker(plugins_list)
   local start_dp = function(premature)
     if premature then
       return
     end
 
-    self.child = require("kong.clustering.wrpc_data_plane").new(self.conf, self.cert, self.cert_key)
+    if kong.configuration.wrpc_hybrid_protocol then
```

Comment:
Proposal/idea: what if we reuse the `legacy_hybrid_protocol` configuration param that was introduced in 3.0? We can simply set the default value to `on`/`true` instead of `off`/`false`?

Reason: it will make it easier to backport any of these changes to the 3.0.x series, if we choose to do so.

Reply:
`legacy_hybrid_protocol` is not suitable for present protocol, which is not legacy anymore. so I use another more clarify flag `wrpc_hybrid_protocol`